app/Http/Controllers/Admin/BookController.php

```diff
@@ -136,18 +120,24 @@ public function index(Request $request)
     /**
      * Show the form with book data for edit book.
      *
-     * @param App\Model\Book $book pass book object
+     * @param App\Model\Book          $book    pass book object
+     * @param Illuminate\Http\Request $request request
      *
      * @return \Illuminate\Http\Response
      */
-    public function edit(Book $book)
+    public function edit(Book $book, Request $request)
     {
         $categoryFields = [
             'id',
             'name'
         ];
+        if (!empty($request->input('page'))) {
```

Comment:
- what's it mean?

Reply:
dạ, nghĩa là để nó nhận cái uri từ bên list, xong nó sẽ truyền sang form như 1 hidden input, nếu đang nhấn edit ở trang 2 thì khi back hoặc submit edit xong sẽ quay về phân trang trước.